pkgs/by-name/hi/highs/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "highs";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "ERGO-Code";
+    repo = "HiGHS";
+    rev = "v${version}";
+    hash = "sha256-Wa5ivUJk0t58FhZD0zy0zxHHj4/p8e9WcxXwu5zenxI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
```

```suggestion

  doCheck = true;
 
```

probably good to enable tests here just as a sanity check

---

Good call, thanks @eclairevoyant